docs/components/rpa/overview.md

```diff
@@ -12,3 +12,5 @@ Camunda uses RPA to facilitate the integration of legacy applications without ac
 Camunda's RPA functionality is built around a standalone RPA worker and [Robot Framework](https://robotframework.org/)-based RPA scripts. This setup allows users to develop and execute RPA scripts that integrate seamlessly with existing Camunda stacks.
```

Comment:
> Robotic Process Automation (RPA) allows you to automate manual, repetitive tasks by interacting with applications directly. Camunda uses RPA to facilitate the integration of legacy applications without accessible APIs into your process orchestration.

What types of applications, for example, are we talking about here?

Reply:
There is a wide range of Applications, such as Legacy Mainframe Applications, Desktop Applications such as ERP systems, or Websites without exposed APIs.